src/core/server/saved_objects/import/check_conflict_for_data_source.test.ts

```diff
@@ -299,4 +316,58 @@ describe('#checkConflictsForDataSource', () => {
       })
     );
   });
+
+  /**
+   * TSVB test cases
+   */
+  it.each([
+    {
+      id: 'some-object-id',
+    },
+    {
+      id: 'old-datasource-id_some-object-id',
+    },
+  ])('will update datasource reference + visState of TSVB visualization', async ({ id }) => {
+    const tsvbSavedObject = createTSVBVisualizationObject(id);
+    // @ts-expect-error
```

Comment:
nit: Let's remove `@ts-expect-error` and handle Type error.

Reply:
Added an internal type `VisualizationObject` to `saved_objects/import`